src/Explorer/MostRecentActivity/MostRecentActivity.ts

```diff
@@ -32,7 +31,7 @@ interface StoredData {
  * Stores most recent activity
  */
 class MostRecentActivity {
-  private static readonly schemaVersion: string = "1";
+  private static readonly schemaVersion: string = "2";
```

Comment:
You might want to do something about the old items saved under version 1. We don't need to maintain support, but we should probably ignore them (so they don't break the code) and maybe even delete them from local storage.

Reply:
I think someone (you?) already handled that. When it gets loaded, if the schema doesn't match, the recent activity is discarded.